include/seqan3/search/fm_index/bi_fm_index.hpp

```diff
@@ -153,7 +143,7 @@ class bi_fm_index
     template <std::ranges::Range text_t>
     bi_fm_index(text_t && text)
```

Comment:
You should now enforce that the text_t's alphabet type is convertible to the alpahbet type of the index, or not?